package.json

```diff
@@ -71,6 +71,7 @@
     "@vitejs/plugin-react": "^1.3.0",
     "jest": "^27.5.1",
     "jest-watch-typeahead": "^1.0.0",
+    "nodemon": "^2.0.15",
```

Comment:
Where do you use `nodemon`? Can you commit that script as well?

Reply:
The `server:watch` script of `jest-preview` use the `nodemon` binary

Reply:
Thanks. I thought `nodemon` is bundled with `node` lol.